packages/lib/src/vis/scatter/ScatterVis.tsx

```diff
@@ -57,7 +57,7 @@ function ScatterVis(props: Props) {
   assertDefined(ordinateDomain, 'Ordinates have undefined domain');
 
   return (
-    <figure className={styles.root}>
+    <figure className={styles.root} aria-label={title} data-keep-canvas-colors>
```

Comment:
Fixing some overlooked attributes...

The `aria-label` is mainly to make tests work. The `data-keep-canvas-colors` fixes the fact that scatter vis were inverted in dark mode up to now.